src/Metrics.tsx

```diff
@@ -1,64 +1,89 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import "twin.macro";
+import ChevronIcon from "./icons/ChevronIcon";
 
 const data = [
-    {
-      name: "Metal Temp",
-      measurement: "56",
-      unit: "℃"
-    },
-    {
-      name: "Pressure",
-      measurement: "0.0",
-      unit: "bar"
-    },
-    {
-      name: "Flow Rate",
-      measurement: "0.0",
-      unit: "ml/s"
-    },
-    {
-      name: "Shot Time",
-      measurement: "0.0",
-      unit: "s"
-    },
-    {
-      name: "Weight",
-      measurement: "0.0",
-      unit: "g"
-    }
+  {
+    name: "Metal Temp",
+    measurement: "56",
+    unit: "℃",
+  },
+  {
+    name: "Pressure",
+    measurement: "0.0",
+    unit: "bar",
+  },
+  {
+    name: "Flow Rate",
+    measurement: "0.0",
+    unit: "ml/s",
+  },
+  {
+    name: "Shot Time",
+    measurement: "0.0",
+    unit: "s",
+  },
+  {
+    name: "Weight",
+    measurement: "0.0",
+    unit: "g",
+  },
```

Comment:
Probably a good idea to decide on a formatting strategy in the future so we can avoid these kind of non-diffs (spaces, commas, etc.)